### PR TITLE
add style on hover over place list and change its marker icon #134

### PR DESCRIPTION
**Pull request by resant18**
##Issue Number:
#134
## Description

##Do any new issues need to be created as a result of this PR"? No
Change cursor to pointer, change background color of place list item, make the selected marker icon bigger when user hover over a place item.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

🚨Please review the [contribution guidelines](https://github.com/gwg-women/gwg-women-techmakers/wiki/Contribution-Guidelines) to this repository.
- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Created a new branch from 'dev' and PR is targeting the project's 'dev' branch.
- [x ] The issue can be closed.

 Provide a descriptive branch name and once the pull request is approved and merged to dev, the branch can be deleted.
styling-placelist-marker branch
💔Thank you!

### CODE REVIEW

**Comment by PepperAddict** [comment]
Wouldn't let me preview due to the conflicts. D:

**Review by digilou — CHANGES_REQUESTED** [review]
@resant18 would you rebase? I did several styling updates last night that got merged.

**Comment by sonalikatara** [comment]
@resant18  do you need help with merging ?

**Comment by sonalikatara** [comment]
@resant18  I noticed you have removed the zoom property from the map and set the initalZoom to 15
`
 removed  zoom={12}
`
`
 added       initialZoom={15}
`
My guess is this was done because we have increased the marker , Have you tested this change by making a couple of different searches like library , coffee, or museum. Do we need to add code to bound the search results in the map after this change ?

**Comment by resant18** [comment]
@sonalikatara @digilou, my changes is not much, I should be able to merge it quickly. In term of initialZoom, I used it for testing the marker icon changes, since it was a bit clustered in some area. I will revert it back for now. In the meantime, we could find a solution, maybe give some condition or even clustered markers.

**Comment by sonalikatara** [comment]
@resant18  do you suggest we increase the zoom level ? what do you think about increasing it to 13 or 14  for now?

**Comment by resant18** [comment]
@sonalikatara I'd say leave it as for now, since in some results, if we increase the zoom level, it will reduce the visible markers due to some of them out of zooming area. Not sure if this is better than overcrowded markers :) What do you think? 

Other issue that I haven't figure it out is how to resize the new marker icon that I applied. It's way too big :) I tried to change the icon's scaledSize, but no luck. Maybe I'm missing something here.

**Comment by sonalikatara** [comment]
`
const selectedIconUrl = 'https://maps.google.com/mapfiles/kml/paddle/red-circle.png';
`
@resant18 

Right now, I can think of two solutions to this issue 

1) save the image in the marker images folder and change it's size, or find a new one

2) look at this documentation [https://developers.google.com/maps/documentation/javascript/examples/icon-complex](https://developers.google.com/maps/documentation/javascript/examples/icon-complex)
They suggest creating an image and setting it's style before adding it to icon. 

I haven't run the pr locally, so don't know id this image is really that big to change, so how about we merge this pr and see what everyone thinks, if we want to change we can create a new issue and change the icon  image.

What do you suggest ?

**Comment by sonalikatara** [comment]
have you taken care of @digilou's suggestion ?

**Comment by resant18** [comment]
@sonalikatara, I agree with you. It's something that we can think about it later. I have merged instead of rebase, since I don't change the code very much. I hope it's okay, @digilou ?

File: src/css/index.scss [hunk 1]
```diff
@@ -112,6 +115,9 @@ h3 {
             font-weight: bold;
             color: var(--header-color);
         }
+        &:hover {
```
Comment (digilou) [inline src/css/index.scss]: Just add `&:focus` after `&:hover` (you can separate by a comma) as best practice for accessibility. Focus gives the same style for keyboard users.

**Comment by sonalikatara** [comment]
Is this PR ready to be merged ? @digilou @resant18 @PepperAddict

**Review by digilou — CHANGES_REQUESTED** [review]

File: src/css/index.scss (hunk 1 above)
Comment (digilou) [reply src/css/index.scss]: Sorry @sonalikatara @resant18, I thought I submitted this request!

Comment (resant18) [reply src/css/index.scss]: @digilou How do I change it? Do I have to create another pull request?

Comment (digilou) [reply src/css/index.scss]: Looks like your changes were pushed!

**Review by digilou — APPROVED** [review]

**Review by PepperAddict — APPROVED** [review]

**Pull request merged**